### PR TITLE
Add curl location option

### PR DESCRIPTION
**Pull request by dozen**
Updated the README. 

`-L` option is required.

Please merge, if no problem. 🙏

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ or
 
 ```
 ## at Linux
-curl -slO https://github.com/fujiwara/nssh/releases/download/v0.0.1/nssh-v0.0.1-linux-amd64.zip
+curl -slLO https://github.com/fujiwara/nssh/releases/download/v0.0.1/nssh-v0.0.1-linux-amd64.zip
```
Comment (fujiwara) [inline README.md]: Sorry, `-l` option was typo of `-L`.
Could you replace `slLO` to `sLO` ?

Comment (dozen) [reply README.md]: OK. I fix it.

**Comment by fujiwara** [comment]
Thanks!

**Comment by dozen** [comment]
🙇

**Pull request merged**